src/main/java/org/spongepowered/api/command/args/parsing/QuotedStringTokenizer.java

```diff
@@ -46,17 +46,15 @@
  * ARGS := ((UNQUOTED_ARG | QUOTED_ARG) WHITESPACE+)+</pre></blockquote>
  */
 class QuotedStringTokenizer implements InputTokenizer {
+    public static final QuotedStringTokenizer INSTANCE = new QuotedStringTokenizer(false);
+    public static final QuotedStringTokenizer LENIENT_INSTANCE = new QuotedStringTokenizer(true);
     private static final int CHAR_BACKSLASH = '\\';
     private static final int CHAR_SINGLE_QUOTE = '\'';
     private static final int CHAR_DOUBLE_QUOTE = '"';
-    private final boolean handleQuotedStrings;
     private final boolean forceLenient;
-    private final boolean trimTrailingSpace;
 
-    QuotedStringTokenizer(boolean handleQuotedStrings, boolean forceLenient, boolean trimTrailingSpace) {
-        this.handleQuotedStrings = handleQuotedStrings;
+    private QuotedStringTokenizer(boolean forceLenient) {
```

Comment:
Leave this package private to avoid creation of synthetic methods please

Reply:
(well, actually, not sure if they would be created in this case, but let's leave it package private)

Reply:
Same thing as with the other two tokenizers, this was the only one that didn't have a private constructor - presumably because it was being called in `InputTokenzier`. I think it's best to be consistent here.

Reply:
Unfortunately, while we’re in the API later, anything destined for a stable API should not have their visibility downgraded, it is part of the API, even if package private. Change it back please.